### ISSUE TITLE
unknown variant `timeless`

### ISSUE THREAD

**Issue by LuckyTurtleDev** [mendess/scryfall-rs#34]
**Describe the bug**
Can not get card info for https://scryfall.com/card/jud/140/phantom-nishoba.

**To Reproduce**
```rust
let id = Uuid::parse_str("56ebc372-aabd-4174-a943-c7bf59e5028d").unwrap();
Card::scryfall_id(id).await.with_context(|| "failed to fetch card informations")?;
```

**Field that failed to deserialise/serde error message**
```
💥 ERROR mtg::cache                     > failed to fetch card informations

Caused by:
    Error making request: error decoding response body: unknown variant `timeless`, expected one of `standard`, `modern`, `legacy`, `vintage`, `commander`, `future`, `pauper`, `pioneer`, `penny`, `duel`, `oldschool`, `historic`, `gladiator`, `brawl`, `premodern`, `historicbrawl`, `paupercommander`, `alchemy`, `explorer`, `predh`, `oathbreaker` at line 1 column 1702
```

**Version:**
0.14.1

**Additional context**
A few days ago it has still work.

**Comment by mendess** [comment]
This new format was introduced yesterday. Will make a new release to fix this soon

**Comment by mendess** [comment]
Fixed in version `0.14.2`

**Comment by LuckyTurtleDev** [comment]
Wow this was fast
